flask_admin/theme.py

```diff
@@ -11,12 +11,10 @@ class Theme:
 
 @dataclass
 class BootstrapTheme(Theme):
-    folder: typing.Literal['bootstrap2', 'bootstrap3', 'bootstrap4']
+    folder: typing.Literal['bootstrap4']
     base_template: str = 'admin/base.html'
     swatch: str = 'default'
     fluid: bool = False
 
 
-Bootstrap2Theme = partial(BootstrapTheme, folder='bootstrap2')
-Bootstrap3Theme = partial(BootstrapTheme, folder='bootstrap3')
 Bootstrap4Theme = partial(BootstrapTheme, folder='bootstrap4')
```

Comment:
What about declaring Bootstrap4Theme without partial?

```suggestion
class Bootstrap4Theme(BootstrapTheme):
    folder='bootstrap4'
```

Reply:
I think, because `BootstrapTheme` is a dataclass, it's preferable to me to just instantiate it than subclass it :)